docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5080` Temporarily fix the double conversion issue for manual latest price.
```

Comment:
Please write user facing changelog entries.

1. Temporarly is wrong. The users don't care.
2. What is the double conversion issue? I am a user. I have no idea what you mean.

```suggestion
* :feature:`5080` In the manual balances section, the should no longer be any double conversion. So 1 euro should always be one euro.
```